src/actions/actions.js

```diff
@@ -3,7 +3,7 @@ import axios from 'axios';
 import { AUTH_USER, UNAUTH_USER, AUTH_ERROR } from './constants'
 
 const instance = axios.create({
-    baseURL: '159.65.69.12:3000',
+    baseURL: 'http://159.65.69.12:3000',
```

Comment:
Hide this please using .gitignored file

Reply:
Refer to a599531c0810059dae07027d3e4fc48ef575997e for changes.